e2e/test/prerequisites/E2ETestsSetup/e2eTestsSetup.ps1

```diff
@@ -321,17 +329,12 @@ Check-AzureCliVersion
 # Install chocolatey and docker
 ########################################################################################################
 
-if ($InstallDependencies)
+if ($InstallDependencies -And $InstallDockerDesktopOnWindows)
 {
-    Write-Host "`nSetting up docker."
```

Comment:
The dev setup process already listed setting up docker on Windows with Chocolatey up-front.
This setup script here also does not work and result in error because docker on windows require reboot after install. This script always fail at this point.

Reply:
Yeah, we had to run it, reboot, then run it again.